src/TodoListItem.js

```diff
@@ -1,9 +1,12 @@
-import React from "react"
+import React from "react";
 
-function TodoListItem({title}) {
-    return(
-        <li>{title}</li>
-    )
+function TodoListItem({ id, title, onRemoveTodo}) {
+  return (
+    <li>
+      {title}
+      <button type="button" onClick= {() => onRemoveTodo(id)}>remove</button>
```

Comment:
let's remove the spacing at `onClick`:

`onClick={() => onRemoveTodo(id)}>remove</button>`